test/samples/sample_split_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rspec"
+
+describe 'SplittableGroup' do
+  it 'runs one test' do
+    sleep(1)
```

Comment:
I do run the tests locally. I like the idea of increasing durations in Travis to improve reliability. I'll do that.